pkg/analytics/tracker.go

```diff
@@ -0,0 +1,11 @@
+package analytics
+
+type AnalyticsTracker struct {
+	TrackFn func(event string, success bool, props map[string]interface{})
```

Comment:
Can this be private?
```suggestion
	trackFn func(event string, success bool, props map[string]interface{})
```